### ISSUE TITLE
Cannot Link Image Carousel Pictures to Their Attachment Files

### ISSUE THREAD

**Issue by favrem** [pojome/elementor#3093]
Cannot link my image carousel pictures to their attachment files, but only to their media file or custom url... If the Pro version is like this too, what is the point?

**Comment by bainternet** [comment]
@favrem 
can you please explain what you call "Attachment files"? and how they are different?

And if you ask about the difference between Elementor And Elementor Pro you can see this [comparison chart ](https://docs.elementor.com/article/146-benefits-elementor-pro)

**Comment by favrem** [comment]
I bought the pro, now its fixed

**Comment by bainternet** [comment]
Still have no idea what was the issue. Anyway, glad it's fixed.

**Comment by favrem** [comment]
Well, if you really want to know what the issue of your client is, ohad.
Here are my issues I’m having with the pro which I have submited two
tickets for. Glad you are so keen to willing to understand. Here it goes.


PROBLEM 1 -
Media Carousel:

In the media carousel, my images do not show up in order. For instance.
Setting #1 for a particular picture will not necessarily set it at #1 on
the actual page, but it will rank it at #4. I realized that its when I have
more than 5 that the order will start messing up.

PROBLEM 2-
Media Carousel:

In the media carousel, again... on some carousels, my images will start
stacking up on each other the more slide to new pictures, instead of just
fading away, they stack on top of each other.

PROBLEM 3-
Media Carousel:

Now this is more of a constructive criticism and feedback for potential
improvements. I have an interior design company, and detail is important.

When I say detail, I mean being able to see the caption and title in the
lightbox. Being able to increase the size of the ‘x’ button to close the
light box, BIGGER. Some old clients won’t see it believe it or not.

And very importantly, have the option to open the image in lightbox, but
also having the option to still navigate through the menues. My clients
will be confused by not seeing anything anymore. This could be done by
making the light box a little smaller. Or just adding the menue on it.

I am confident this will be looke through thoroughly for the benefit of
your client base.

Thank you,
Mehdi Favre
